src/ObjectUtils/PdbFileLlvm.cpp

```diff
@@ -58,9 +61,10 @@ class SymbolInfoVisitor : public llvm::codeview::SymbolVisitorCallbacks {
 
     // The address in PDB files is a relative virtual address (RVA), to make the address compatible
     // with how we do the computation, we need to add both the load bias (ImageBase for COFF) and
-    // the offset of the executable section.
-    symbol_info.set_address(proc.CodeOffset + object_file_info_.load_bias +
-                            object_file_info_.executable_segment_offset);
+    // the offset of the section.
+    ORBIT_CHECK(section_headers_ != nullptr && section_headers_->size() >= proc.Segment);
+    uint64_t section_offset = (*section_headers_)[proc.Segment - 1].VirtualAddress;
```

Comment:
I added a comment and the additional check. My understanding is that this is purely because the segments starts counting at 1. I've run this code with LLVM enforced with a bunch of Windows pdb's and the check did not failed.